### PR TITLE
Added Pie and Q text

### PR DESCRIPTION
**Pull request by TipzTeam**

### CODE REVIEW

File: sdat2img.py [hunk 1]
```diff
@@ -74,7 +74,7 @@ def parse_transfer_list_file(path):
     elif version == 3:
         print('Android Marshmallow 6.x detected!\n')
     elif version == 4:
-        print('Android Nougat 7.x / Oreo 8.x detected!\n')
+        print('Android Nougat 7.x / Oreo 8.x / Pie 9.x / Q 10.x detected!\n')
```
Comment (imknown) [inline sdat2img.py]: It should be `Pie 9 / Q 10`

Comment (TipzTeam) [reply sdat2img.py]: ok

Comment (TipzTeam) [reply sdat2img.py]: fixed it, android p and q tested working

**Comment by xpirt** [comment]
This will need further testing before merging. 

Closed.

**Pull request closed without merging**